### PR TITLE
Add text mentioning replacement integration.

### LINKED ISSUE #22018: Updater deprecation: why and alternative?
### Feedback

Is says it's deprecated, but further explanation is given. Some background information and an alternative would be great.

### URL

https://www.home-assistant.io/integrations/updater/

### Version

2022.3.4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by skipishere**
## Proposed change
Adds some information about using the Version integration to replace similar functionality that has been depriciated.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #22018 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/updater.markdown [hunk 1]
```diff
@@ -16,7 +16,8 @@ ha_platforms:
 <div class='note warning'>
 
 This integration is deprecated and will be removed in Home Assistant Core 2022.5.
-
+  
+Similar functionality is available using the [Version integration](https://www.home-assistant.io/integrations/version/)
```
Comment (frenck) [inline source/_integrations/updater.markdown]:
```suggestion
Similar functionality is available using the [Version integration](/integrations/version/)
```

File: source/_integrations/updater.markdown [hunk 2]
```diff
@@ -16,7 +16,8 @@ ha_platforms:
 <div class='note warning'>
 
 This integration is deprecated and will be removed in Home Assistant Core 2022.5.
-
+  
+Similar functionality is available using the [Version integration](/integrations/version/)
```
Comment (frenck) [inline source/_integrations/updater.markdown]: New line to fix Parser context switching

```suggestion
Similar functionality is available using the [Version integration](/integrations/version/)

```

File: source/_integrations/updater.markdown [hunk 3]
```diff
@@ -16,6 +16,8 @@ ha_platforms:
 <div class='note warning'>
 
 This integration is deprecated and will be removed in Home Assistant Core 2022.5.
+  
+Similar functionality is available using the [Version integration](/integrations/version/)
```
Comment (frenck) [inline source/_integrations/updater.markdown]:
```suggestion
Similar functionality is available using the [Version integration](/integrations/version/).
```

**Review by frenck — APPROVED** [review]
Thanks, @skipishere 👍

File: source/_integrations/updater.markdown [hunk 4]
```diff
@@ -16,6 +16,8 @@ ha_platforms:
 <div class='note warning'>
 
 This integration is deprecated and will be removed in Home Assistant Core 2022.5.
+  
+Similar functionality is available using the [Version integration](/integrations/version/).
```
Comment (ludeeus) [inline source/_integrations/updater.markdown:20]: This is only partially true.
What Version integration provides is only part of what the updater integration did.
The Analytics integration has taken over the reporting part of the updater integration.

Comment (frenck) [reply source/_integrations/updater.markdown:20]: That is the only part there for quite some time, and it is what is causing the breaking change at this point. So, I guess this is fine/enough to have at this point.

Comment (skipishere) [reply source/_integrations/updater.markdown:20]: To be honest I wasn't sure how much detail to put in to that. Happy to add more around that if needed/wanted but I'll admit I'd be copy/pasting from the release notes mentioning it being removed given I don't know the context of the analytics part

**Pull request merged**